typings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "ambientDependencies": {
```

Comment:
you're also missing the `ng2` type info 
https://github.com/gdi2290/typings-ng2
as
`"ng2": "github:gdi2290/typings-ng2/ng2.d.ts#32998ff5584c0eab0cd9dc7704abb1c5c450701c",`